tools/Dashboard/DevHome.Dashboard/Services/IWidgetScreenshotService.cs

```diff
@@ -10,5 +10,7 @@ namespace DevHome.Dashboard.Services;
 
 public interface IWidgetScreenshotService
 {
-    public Task<BitmapImage> GetScreenshotFromCache(WidgetDefinition widgetDefinition, ElementTheme actualTheme);
+    public void RemoveScreenshotsFromCache(string definitionId);

```

Comment:
Typo RemoveScreenshot`s`FromCache.

Reply:
Added on purpose, since it's maybe removing two screenshots... maybe I'll reconsider in 2482